frontend/test/metabase-db/mongo/query.cy.spec.js

```diff
@@ -97,6 +97,9 @@ function writeNativeMongoQuery() {
   cy.findByText("Native query").click();
   cy.findByText(MONGO_DB_NAME).click();
 
+  cy.findByText("Select a table").click();
+  cy.findByText("Orders").click();
```

Comment:
prefetching made it so that this table gets automatically selected. the logic for that, unfortunately, is sequenced alongside the selection of the DB, so it won't happen the first time around, until the table values are cached